conans/test/functional/toolchains/cmake/test_cmake.py

```diff
@@ -15,6 +15,33 @@
 from conans.util.files import save
 
 
+@pytest.mark.tool_mingw64
+@pytest.mark.tool_cmake(version="3.15")
+@pytest.mark.skipif(platform.system() != "Windows", reason="Needs windows")
+def test_simple_cmake_mingw():
+    client = TestClient()
+    client.run("new hello/1.0 -m cmake_lib")
+    client.save({"mingw": """
+
+    [settings]
+    os=Windows
+    arch=x86_64
+    build_type=Release
+    compiler=gcc
+    compiler.exception=seh
+    compiler.libcxx=libstdc++11
+    compiler.threads=win32
+    compiler.version=11.2
+    cppstd=17
+    [options]
+    [build_requires]
+    [env]
+    CC=gcc
+    CXX=g++
+    """})
+    client.run("create . --profile=mingw")
```

Comment:
The template include output of compiler version (which is now not well defined not checked, the compiler in CI might not be 11.2, and it won't be noticed). But checking som eof the binary output like ``__GNUC__`` and that it doesn't contain any trace of Visual Studio, would be good